lib/Transforms/MemoryBanking.cpp

```diff
@@ -71,16 +71,21 @@ DenseSet<Value> collectMemRefs(mlir::affine::AffineParallelOp parOp) {
   return memrefVals;
 }
 
-MemRefType computeBankedMemRefType(MemRefType originalType,
-                                   uint64_t bankingFactor,
-                                   unsigned bankingDimension) {
-  ArrayRef<int64_t> originalShape = originalType.getShape();
+// Verify the banking scheme with different conditions.
+void verifyBankingConditions(unsigned bankingDimension, unsigned bankingFactor,
+                             MemRefType originalType) {
+  auto originalShape = originalType.getShape();
```

Comment:
Suppppper nit: any reason why you got rid of the type here
```suggestion
  ArrayRef<int64_t> originalShape = originalType.getShape();
```